packages/web-react/src/components/Text/stories/TextSizes.tsx

```diff
@@ -8,6 +8,7 @@ const Story = (props: unknown) => (
     <Text size="large">Body Large Text Regular</Text>
```

Comment:
![Screenshot 2023-02-01 at 10 02 48](https://user-images.githubusercontent.com/2481010/215998241-a300a0f8-3246-4da3-9e50-299669f8227b.png)
Should you order those sizes in the story, please?

Reply:
Should it be sorted by size only in story or everywhere? Currently, we mix ABC and size/scale sorting :(